index.js

```diff
@@ -1,13 +1,43 @@
-const mongoose = require('mongoose');
-const Recipe = require('./models/Recipe.model'); // Import of the model Recipe from './models/Recipe.model.js'
-const data = require('./data.js'); // Import of the data from './data.js'
+const mongoose = require("mongoose");
+const Recipe = require("./models/Recipe.model"); // Import of the model Recipe from './models/Recipe.model.js'
+const data = require("./data.js"); // Import of the data from './data.js'
 
 // Connection to the database "recipeApp"
 mongoose
-  .connect('mongodb://localhost/recipe-app-dev', {
+  .connect("mongodb://localhost/recipe-app-dev", {
     useCreateIndex: true,
     useNewUrlParser: true,
     useUnifiedTopology: true
   })
-  .then(x => console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`))
-  .catch(err => console.error('Error connecting to mongo', err));
+  .then(x =>
+    console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`)
+  )
+  .catch(err => console.error("Error connecting to mongo", err));
+
+Recipe.create({
+  title: "Pizza",
+  level: "UltraPro Chef",
+  ingredients: ["tomatos", "cheese", "olives"],
+  cuisine: "italian",
+  dishType: "Dish",
+  duration: 10,
+  creator: "Markus"
+}).then(created => {
+  console.log(created);
+});
+
+Recipe.insertMany(data).then(inserted => {
+  console.log(inserted);
+});
+
+Recipe.updateOne({ title: "Rigatoni alla Genovese" }, { duration: 100 }).then(
+  result => {
+    console.log(result);
+  }
+);
+
+Recipe.deleteOne({ title: "Carrot Cake" }).then(deleted => {
+  console.log(deleted);
+});
+
+mongoose.connection.close();
```

Comment:
carfeul with closing the connection in the global scope.
Since all of this functions are asyncronous the close() method can happen before the other methods such as create, delete or update.
This will create a lot of problems becuase your connection  could close before any of the other methods are done so it will give you an error such as this
```
UnhandledPromiseRejectionWarning: MongoError: server is closed
    at executeWriteOperation (/private/tmp/lab-mongoose-recipes/node_modules/mongodb/lib/core/sdam/server.js:470:14)
```